tensorflow/python/framework/test_util.py

```diff
@@ -473,7 +473,9 @@ def assertAllClose(self, a, b, rtol=1e-6, atol=1e-6):
       print("dtype = %s, shape = %s" % (a.dtype, a.shape))
       np.testing.assert_allclose(a, b, rtol=rtol, atol=atol)
 
-  def assertAllCloseAccordingToType(self, a, b, rtol=1e-6, atol=1e-6):
+  def assertAllCloseAccordingToType(self, a, b, rtol=1e-6, atol=1e-6,
```

Comment:
Please add a unit test for this function in /tensorflow/python/framework/tensor_util_test.py.